### DISCUSSION TITLE
UI翻訳はV2.0では廃止されたのでしょうか？(UI翻译在V2.0中已经停止了吗？Has UI translation been discontinued in V2.0?)

### DISCUSSION THREAD

**Discussion by namajake29** [the1812/Bilibili-Evolved#2654] [提问 / Q&A]
日本語で失礼します。
Bilibili-Evolvedを使用してBiliBiliを観ています。
V2.0に更新したところUI翻訳がなくなっていました。
在线仓库を探してみたりしましたがUI翻訳は見つけられませんでした。
V2.0ではUI翻訳は廃止されたのでしょうか？

(请原谅机器翻译我用Bilibili-Evolved看BiliBili。我已经更新到V2.0，但UI翻译却不见了。我试图搜索在线仓库，但找不到UI翻译。在V2.0中，UI翻译不再可用了吗？

Please excuse the machine translation.
I'm using Bilibili-Evolved to watch BiliBili.
I just updated to V2.0 and the UI translation is gone.
I tried searching 在线仓库 but could not find the UI translation.
Has the UI translation been discontinued in V2.0?)

**Comment by the1812** [comment]
Yes. The Japanese language translation provider @Coulomb-G had stopped contributing since v1.12.16 (2021.08.21, #2180)
We are planning to add English support in the future.

**Reply by the1812** [reply]
The last v1 version can be downloaded from https://github.com/the1812/Bilibili-Evolved/raw/master/bilibili-evolved.offline.user.js

**Reply by namajake29** [reply]
thank you
